src/__tests__/footer.test.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-undef */
+import { render, screen, cleanup } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Footer from "../components/Footer";
+import React from "react";
+import { configureStore } from "@reduxjs/toolkit";
+import rootReducer from "../reducers/index";
+import { Provider } from "react-redux";
+
+afterEach(() => {
```

Comment:
This block needs to be removed as well, since no jest mocks being used.

Reply:
Fixed